config/eo.yml

```diff
@@ -4,12 +4,13 @@ idspace: EO
 base_url: /obo/eo
 
 products:
-- eo.owl: http://www.berkeleybop.org/ontologies/eo.owl
+- eo.owl: https://raw.githubusercontent.com/Planteome/plant-environment-ontology/master/plant-environment-ontology.obo.owl
+- eo.OBO: https://raw.githubusercontent.com/Planteome/plant-environment-ontology/master/plant-environment-ontology.obo
 
 term_browser: ontobee
 example_terms:
 - EO_0007359
 
 entries:
 - exact: /tracker
-  replacement: https://sourceforge.net/tracker/?atid=5010796&group_id=76834&func=browse
+- replacement: https://github.com/Planteome/plant-environment-ontology/issues
```

Comment:
Ok, will change, but I just copied and pasted what was in the file previously, then replaced the link to the tracker....